colcon_ros/task/ros.py

```diff
@@ -59,6 +59,7 @@ def add_arguments(self, *, parser):  # noqa: D102
                 'share/{pkg.name}/local_setup.bat'.format_map(locals()),
                 'share/{pkg.name}/local_setup.ps1'.format_map(locals()),
                 'share/{pkg.name}/local_setup.sh'.format_map(locals()),
+                'share/{pkg.name}/local_setup.zsh'.format_map(locals()),
```

Comment:
Since this package doesn't depend on `zsh` I would rather not hard code this here. The same problem is already the case for `bash`.

See #29 for an alternative approach.

Reply:
Yeah I agree it didn't seem appropriate that this package was hard coding without depending on the shell specific packages (like colcon-powershell or colcon-zsh)